control_protocol.md

```diff
@@ -318,6 +318,18 @@ For the format of the Messages, see {ref}`control_protocol.md#message-layer`.
 
 ## Message layer
 
+The message layer contains the actual information exchanged between Components.
+As LECO is about controlling experiments, the message layer has to transmit commands, that is calling procedures remotely.
+
+We use the [JSON-RPC](https://www.jsonrpc.org/specification) standard to encode these *remote procedure calls* (RPC) and the responses.
+We further use the [OpenRPC](https://open-rpc.org/) standard to describe the possibly callable methods of a Component.
+
+Therefore, a Component MUST execute remote procedures according to JSON-RPC and return an appropriate response.
+A Component MUST also offer a list of all possibly callable methods in accordance with OpenRPC.
+
+For such a RPC message, the first content frame MUST consist in a JSON-RPC compatible content, for example a single request object or a batch of request objects.
```

Comment:
```suggestion
For such a RPC message, the content frame MUST consist of a JSON-RPC compatible content, for example a single request object or a batch of request objects.
```

Reply:
Here we have to distinguish the message layer (where more than one frame is allowed) and the content layer, where only one frame is allowed for our default implementation.
A possible binary implementation or something from a user might include more than one frame.

That's the reasoning behind my formulation.

Reply:
Ok, I understand, that is okay for me.